staging/src/k8s.io/apiserver/pkg/endpoints/handlers/get.go

```diff
@@ -67,9 +67,11 @@ func getResourceHandler(scope RequestScope, getter getterFunc) http.HandlerFunc
 			scope.err(fmt.Errorf("missing requestInfo"), w, req)
 			return
 		}
-		if err := setSelfLink(result, requestInfo, scope.Namer); err != nil {
-			scope.err(err, w, req)
-			return
+		if req.URL.Query().Get("export") != "true" {
```

Comment:
How about adding "exports bool" to the return value of the `getterFunc`, since the query parameters are [handled](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/get.go#L88) there formally.